drivers/net/Kconfig

```diff
@@ -485,4 +485,9 @@ config FUJITSU_ES
 
 source "drivers/net/hyperv/Kconfig"
 
+config XAPTUM_PSOCK
```

Comment:
This should go in `drivers/usb/gadget/KConfig` too.

Looks like the convention to split the config option used to enabling building the function (e.g., `UBS_F_ECM`) and the actual config option selected by the user (e.g., USB_CONFIGFS_ECM). The automatically selects the former.

 So
1) add a `USB_F_XAPTUM_PSOCK` to [this list](https://elixir.bootlin.com/linux/latest/source/drivers/usb/gadget/Kconfig#L216)
2) and add a `USB_CONFIGFS_XAPTUM_PSOCK` to [this list](https://elixir.bootlin.com/linux/latest/source/drivers/usb/gadget/Kconfig#L472).

Or, add both to a new file called `drivers/usb/gadget/xaptum/KConfig` and source that file [here](https://elixir.bootlin.com/linux/latest/source/drivers/usb/gadget/Kconfig#L512).

Reply:
I added both defs to the file and added USB_CONFIGFS as a dependency.